### PR TITLE
EDA Localization - Spring '21 drop 1 complete

### PR DESCRIPTION
**Pull request by dmkud**
# Critical Changes

Translated labels for SPring '21 drop 1

# Changes

# Issues Closed

# New Metadata

## Unpackaged Metadata

# Deleted Metadata

# Testing Notes

Please check the correct Male/Female labels were left in English

### CODE REVIEW

File: src/translations/nl_NL.translation [hunk 1]
```diff
@@ -1,1361 +1,1361 @@
...
     </customLabels>
     <customLabels>
-        <label>Huishouden</label>
         <name>DefaultHouseholdName</name>
+        <label>Huishouden</label>
     </customLabels>
     <customLabels>
-        <label>Vrouw</label>
         <name>Female</name>
+        <label>Vrouw</label>
```
Comment (MatthewBlanski) [inline src/translations/nl_NL.translation:49]: @dmkud This is a system value and should not be translated
```suggestion
        <label>Female</label>
```

File: src/translations/ca.translation [hunk 2]
```diff
@@ -1,1359 +1,1359 @@
...
     </customLabels>
     <customLabels>
-        <label>Llar</label>
         <name>DefaultHouseholdName</name>
+        <label>Llar</label>
     </customLabels>
     <customLabels>
-        <label>Dona</label>
         <name>Female</name>
+        <label>Dona</label>
```
Comment (MatthewBlanski) [inline src/translations/ca.translation:49]: @dmkud This is a system value and should not be translated
```suggestion
        <label>Female</label>
```

File: src/translations/de.translation [hunk 3]
```diff
@@ -1,1361 +1,1361 @@
...
     </customLabels>
     <customLabels>
-        <label>Haushalt</label>
         <name>DefaultHouseholdName</name>
+        <label>Haushalt</label>
     </customLabels>
     <customLabels>
-        <label>Weiblich</label>
         <name>Female</name>
+        <label>Weiblich</label>
```
Comment (MatthewBlanski) [inline src/translations/de.translation:49]: @dmkud This is a system value and should not be translated
```suggestion
        <label>Female</label>
```

File: src/translations/es.translation [hunk 4]
```diff
@@ -1,1359 +1,1359 @@
...
     </customLabels>
     <customLabels>
-        <label>Hogar</label>
         <name>DefaultHouseholdName</name>
+        <label>Hogar</label>
     </customLabels>
     <customLabels>
-        <label>Femenino</label>
         <name>Female</name>
+        <label>Femenino</label>
```
Comment (MatthewBlanski) [inline src/translations/es.translation:49]: @dmkud This is a system value and should not be translated
```suggestion
        <label>Female</label>
```

File: src/translations/es_MX.translation [hunk 5]
```diff
@@ -1,1359 +1,1359 @@
...
     </customLabels>
     <customLabels>
-        <label>Hogar</label>
         <name>DefaultHouseholdName</name>
+        <label>Hogar</label>
     </customLabels>
     <customLabels>
-        <label>Femenino</label>
         <name>Female</name>
+        <label>Femenino</label>
```
Comment (MatthewBlanski) [inline src/translations/es_MX.translation:49]: @dmkud This is a system value and should not be translated
```suggestion
        <label>Female</label>
```

File: src/translations/fi.translation [hunk 6]
```diff
@@ -1,1361 +1,1361 @@
...
     </customLabels>
     <customLabels>
-        <label>Kotitalous</label>
         <name>DefaultHouseholdName</name>
+        <label>Kotitalous</label>
     </customLabels>
     <customLabels>
-        <label>Nainen</label>
         <name>Female</name>
+        <label>Nainen</label>
```
Comment (MatthewBlanski) [inline src/translations/fi.translation:49]: @dmkud This is a system value and should not be translated
```suggestion
        <label>Female</label>
```

File: src/translations/fr.translation [hunk 7]
```diff
@@ -1,1359 +1,1359 @@
...
     </customLabels>
     <customLabels>
-        <label>Foyer</label>
         <name>DefaultHouseholdName</name>
+        <label>Foyer</label>
     </customLabels>
     <customLabels>
-        <label>Féminin</label>
         <name>Female</name>
+        <label>Féminin</label>
```
Comment (MatthewBlanski) [inline src/translations/fr.translation:49]: @dmkud This is a system value and should not be translated
```suggestion
        <label>Female</label>
```

File: src/translations/ja.translation [hunk 8]
```diff
@@ -1,1361 +1,1361 @@
...
     </customLabels>
     <customLabels>
-        <label>世帯</label>
         <name>DefaultHouseholdName</name>
+        <label>世帯</label>
     </customLabels>
     <customLabels>
-        <label>女性</label>
         <name>Female</name>
+        <label>女性</label>
```
Comment (MatthewBlanski) [inline src/translations/ja.translation:49]: @dmkud This is a system value and should not be translated
```suggestion
        <label>Female</label>
```

File: src/translations/ca.translation [hunk 9]
```diff
@@ -1,1359 +1,1359 @@
...
     </customLabels>
     <customLabels>
-        <label>El camp introduït com a camp de filtre no és vàlid. Els camps de filtre poden ser qualsevol camp estàndard o personalitzat de l&apos;objecte per al qual s&apos;activen, així com camps dels registres relacionats. Assegureu-vos d&apos;utilitzar el nom de l&apos;API del camp i ometeu qualsevol prefix d&apos;espai de noms EDA (&quot;hed__&quot;).</label>
         <name>InvalidFilter</name>
+        <label>El camp introduït com a camp de filtre no és vàlid. Els camps de filtre poden ser qualsevol camp estàndard o personalitzat de l'objecte per al qual s'activen, així com camps dels registres relacionats. Assegureu-vos d'utilitzar el nom de l'API del camp i ometeu qualsevol prefix d'espai de noms EDA ("hed__").</label>
     </customLabels>
     <customLabels>
-        <label>Home</label>
         <name>Male</name>
+        <label>Home</label>
```
Comment (MatthewBlanski) [inline src/translations/ca.translation:57]: @dmkud This is a system value and should not be translated
```suggestion
        <label>Male</label>
```

File: src/translations/de.translation [hunk 10]
```diff
@@ -1,1361 +1,1361 @@
...
     </customLabels>
     <customLabels>
-        <label>Das als Filterfeld eingegebene Feld ist nicht gültig. Bei Filterfeldern kann es sich um jedes Standard- oder benutzerdefinierte Feld des Objekts handeln, für das dieser Auslöser gilt, sowie um Felder in verwandten Datensätzen. Stellen Sie sicher, dass Sie den API-Namen des Feldes verwenden, und lassen Sie EDA-Namespace-Präfixe (&quot;hed__&quot;) weg.</label>
         <name>InvalidFilter</name>
+        <label>Das als Filterfeld eingegebene Feld ist nicht gültig. Bei Filterfeldern kann es sich um jedes Standard- oder benutzerdefinierte Feld des Objekts handeln, für das dieser Auslöser gilt, sowie um Felder in verwandten Datensätzen. Stellen Sie sicher, dass Sie den API-Namen des Feldes verwenden, und lassen Sie EDA-Namespace-Präfixe ("hed__") weg.</label>
     </customLabels>
     <customLabels>
-        <label>Männlich</label>
         <name>Male</name>
+        <label>Männlich</label>
```
Comment (MatthewBlanski) [inline src/translations/de.translation:57]: @dmkud This is a system value and should not be translated
```suggestion
        <label>Male</label>
```

File: src/translations/es.translation [hunk 11]
```diff
@@ -1,1359 +1,1359 @@
...
     </customLabels>
     <customLabels>
-        <label>El campo introducido como el Campo de filtro no es válido. Los campos de filtro pueden ser cualquier campo estándar o personalizado en el objeto para el que es este desencadenador, así como campos en registros relacionados. Asegúrese de que utiliza el nombre de API del campo, y omite cualquier prefijo de espacio de nombres de EDA (&quot;hed__&quot;).</label>
         <name>InvalidFilter</name>
+        <label>El campo introducido como el Campo de filtro no es válido. Los campos de filtro pueden ser cualquier campo estándar o personalizado en el objeto para el que es este desencadenador, así como campos en registros relacionados. Asegúrese de que utiliza el nombre de API del campo, y omite cualquier prefijo de espacio de nombres de EDA ("hed__").</label>
     </customLabels>
     <customLabels>
-        <label>Masculino</label>
         <name>Male</name>
+        <label>Masculino</label>
```
Comment (MatthewBlanski) [inline src/translations/es.translation:57]: @dmkud This is a system value and should not be translated
```suggestion
        <label>Male</label>
```

File: src/translations/fi.translation [hunk 12]
```diff
@@ -1,1361 +1,1361 @@
...
     </customLabels>
     <customLabels>
-        <label>Suodatinkentäksi syötetty kenttä ei kelpaa. Suodatinkentät voivat olla mitä tahansa vakio- tai mukautettuja kenttiä objektissa, jolle käynnistin on tarkoitettu, samoin kuin kentät liittyvissä tietueissa. Varmista, että käytät kentän API-nimeä ja jätä pois EDA-nimitilan etuliite (&quot;hed__&quot;).</label>
         <name>InvalidFilter</name>
+        <label>Suodatinkentäksi syötetty kenttä ei kelpaa. Suodatinkentät voivat olla mitä tahansa vakio- tai mukautettuja kenttiä objektissa, jolle käynnistin on tarkoitettu, samoin kuin kentät liittyvissä tietueissa. Varmista, että käytät kentän API-nimeä ja jätä pois EDA-nimitilan etuliite ("hed__").</label>
     </customLabels>
     <customLabels>
-        <label>Mies</label>
         <name>Male</name>
+        <label>Mies</label>
```
Comment (MatthewBlanski) [inline src/translations/fi.translation:57]: @dmkud This is a system value and should not be translated
```suggestion
        <label>Male</label>
```

File: src/translations/fr.translation [hunk 13]
```diff
@@ -1,1359 +1,1359 @@
...
     </customLabels>
     <customLabels>
-        <label>Le champ saisi dans le Champ de filtrage n&apos;est pas valide. Les champs de filtrage peuvent correspondre à n&apos;importe quel champ standard ou personnalisé dans l&apos;objet auquel ce déclencheur est destiné, ainsi qu&apos;à des champs dans des enregistrements associés. Assurez-vous d&apos;utiliser le nom d&apos;API du champ sans insérer le préfixe d&apos;espace de noms EDA (&quot;hed__&quot;).</label>
         <name>InvalidFilter</name>
+        <label>Le champ saisi dans le Champ de filtrage n'est pas valide. Les champs de filtrage peuvent correspondre à n'importe quel champ standard ou personnalisé dans l'objet auquel ce déclencheur est destiné, ainsi qu'à des champs dans des enregistrements associés. Assurez-vous d'utiliser le nom d'API du champ sans insérer le préfixe d'espace de noms EDA ("hed__").</label>
     </customLabels>
     <customLabels>
-        <label>Masculin</label>
         <name>Male</name>
+        <label>Masculin</label>
```
Comment (MatthewBlanski) [inline src/translations/fr.translation:57]: @dmkud This is a system value and should not be translated
```suggestion
        <label>Male</label>
```

File: src/translations/ja.translation [hunk 14]
```diff
@@ -1,1361 +1,1361 @@
...
     </customLabels>
     <customLabels>
-        <label>[検索条件項目] として入力された項目が無効です。[検索条件項目] には、このトリガの対象であるオブジェクトの標準またはカスタム項目のほか、関連レコードの項目を使用できます。項目の API 参照名を使用していることを確認し、EDA 名前空間プレフィックス (「hed__」) を除外してください。</label>
         <name>InvalidFilter</name>
+        <label>[検索条件項目] として入力された項目が無効です。[検索条件項目] には、このトリガの対象であるオブジェクトの標準またはカスタム項目のほか、関連レコードの項目を使用できます。項目の API 参照名を使用していることを確認し、EDA 名前空間プレフィックス (「hed__」) を除外してください。</label>
     </customLabels>
     <customLabels>
-        <label>男性</label>
         <name>Male</name>
+        <label>男性</label>
```
Comment (MatthewBlanski) [inline src/translations/ja.translation:57]: @dmkud This is a system value and should not be translated
```suggestion
        <label>Male</label>
```

File: src/translations/nl_NL.translation [hunk 15]
```diff
@@ -1,1361 +1,1361 @@
...
     </customLabels>
     <customLabels>
-        <label>Het veld dat als filterveld is ingevoerd is ongeldig. Filtervelden kunnen elk standaard of aangepast veld zijn op het object waarvoor deze trigger is, evenals een veld op een gerelateerde record. Zorg dat u de API-naam van het veld gebruikt en laat EDA-naamruimtevoorvoegsels (&quot;hed__&quot;) weg.</label>
         <name>InvalidFilter</name>
+        <label>Het veld dat als filterveld is ingevoerd is ongeldig. Filtervelden kunnen elk standaard of aangepast veld zijn op het object waarvoor deze trigger is, evenals een veld op een gerelateerde record. Zorg dat u de API-naam van het veld gebruikt en laat EDA-naamruimtevoorvoegsels ("hed__") weg.</label>
     </customLabels>
     <customLabels>
-        <label>Man</label>
         <name>Male</name>
+        <label>Man</label>
```
Comment (MatthewBlanski) [inline src/translations/nl_NL.translation:57]: @dmkud This is a system value and should not be translated
```suggestion
        <label>Male</label>
```

**Review by MatthewBlanski — CHANGES_REQUESTED** [review]
@dmkud Some of the labels that should not be translated were translated. Please confirm.

File: src/translations/ja.translation (hunk 8 above)
Comment (MatthewBlanski) [reply src/translations/ja.translation:49]: @dmkud This was not captured in the changes.

**Review by MatthewBlanski — APPROVED** [review]

**Comment by spinkelman** [comment]
After speaking with Matt, this passes QA. Merging and Closing

**Pull request merged**